cmd/apiProxyUpload_test.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunApiProxyUpload(t *testing.T) {
```

Comment:
I'd expect more test cases then "good and bad". what should happen if
- file not existing
- file not accessible
- large file 
- possible and unexpected status codes
- connection hangs
- credentials invalid

Reply:
Its not about the file mock utility its about the behavior of ApiProxyUpload.
what does ApiProxyUpload do, if the file does not exist?
what does ApiProxyUpload do, if the file != zip?
Tests should explain this and make sure that it doesn't change.
Without tests i have to look very deep into your code to understand and to review.

Reply:
added new tests

Reply:
@rodibrin Do the tests help you better now to understand the code?